src/site/content/en/patterns/web-vitals-patterns/index.md

```diff
@@ -1,8 +1,6 @@
 ---
 layout: pattern-set
 title: Web Vitals patterns
-description: A collection of code snippets to help you optmize your web projects.
-hero: image/tcFciHGuF3MxnTr1y5ue01OGLBn2/dYAbN2LdttJ1qYk1gssh.svg
+description: UX patterns are a frequent source of Core Web Vitals issues. These code samples show how to implement common UX patterns in a way that is Core Web Vitals friendly.
```

Comment:
I think we should rephrase this to something like:

"A collection of common UX patterns optimized for Core Web Vitals. This collection includes patterns that are often tricky to implement without hurting your Core Web Vitals scores. You can use the code in these examples to help ensure your projects stay on the right track."